pyproject.toml

```diff
@@ -97,19 +97,38 @@ raw-options = { version_scheme = "release-branch-semver" }
 [tool.hatch.build.hooks.vcs]
 version-file = "fury/_version.py"
 
-[tool.blue]
+[tool.ruff]
 line_length = 88
 target-version = ["py38"]
-force-exclude = """
-(
-  _version.py
-  *docs/experimental/*
-)
-"""
-
-[flake8]
-max-line-length = 88
-extend-exclude = ["_version.py", "externals", "*docs/experimental*"]
+exclude = [
+    ".git",
+    "__pycache__",
+    "build",
+    "dist",
+    "_version.py",
```

Comment:
docs/experimental to add

Reply:
Same comment as above :point_up:.

Reply:
Cross-referencing https://github.com/fury-gl/fury/pull/881#discussion_r1585605369. Excluding `docs/experimental`. Resolving.